platform/golden/cfg/preseed-debian-11.cfg

```diff
@@ -50,15 +51,26 @@ tasksel tasksel/skip-tasks  string standard
 tasksel tasksel/first multiselect openssh-server
 popularity-contest popularity-contest/participate boolean false
 
-d-i pkgsel/include string curl wget openssh-server neovim sudo cloud-init
+d-i pkgsel/include string curl wget openssh-server neovim sudo cloud-init ovmf grub-efi-amd64
 d-i pkgsel/language-pack-patterns   string
 d-i pkgsel/install-language-support boolean false
 d-i pkgsel/upgrade select full-upgrade
 
+# -------- Hardware Firmware --------
+d-i hw-detect/load_firmware boolean true
+
 ###########
 ## Use XFS
 
+# Force UEFI booting ('BIOS compatibility' will be lost). Default: false.
```

Comment:
Salsa is Debian's installer. Would [you read this](https://salsa.debian.org/installer-team/partman-efi/-/blob/master/debian/partman-efi.templates#L48-59) and tell me what you make of it?